cmd/siac/main.go

```diff
@@ -86,12 +179,34 @@ func main() {
 				Cmd: walletCmd,
 				Sub: []flagg.Tree{
 					{Cmd: walletBalanceCmd},
+					{Cmd: walletAddressCmd},
+					{Cmd: walletAddressesCmd},
+					{Cmd: walletTransactionsCmd},
+					{Cmd: walletSignCmd},
+					{Cmd: walletBroadcastCmd},
+				},
+			},
+			{
+				Cmd: txpoolCmd,
+				Sub: []flagg.Tree{
+					{Cmd: txpoolTransactionsCmd},
+				},
+			},
+			{
+				Cmd: syncerCmd,
+				Sub: []flagg.Tree{
+					{Cmd: syncerPeersCmd},
+					{Cmd: syncerAddressCmd},
+					{Cmd: syncerConnectCmd},
 				},
 			},
 		},
 	})
 	args := cmd.Args()
 
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+	defer w.Flush()
```

Comment:
A tabwriter is useful for multiple commands, but it should be initialized separately within each command's scope. We can define a helper function though, so that we don't need to copy-paste the same magic arguments each time:
```go
func makeTabWriter() *tabwriter.Writer {
    return tabwriter.NewWriter(
        os.Stdout,
        0,   // minwidth: zero, since we're left-aligning
        0,   // tabwidth: zero, since we're using ' ',  not '\t'
        2,   // padding:  two, so that columns aren't right next to each other
        ' ', // padchar:  spaces, not tabs
        0,   // flags:    none
    )
}
```